queries.sql

```diff
@@ -8,3 +8,56 @@ SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+
+BEGIN;
+
+
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals WHERE species = 'unspecified'; 
+
+
+ROLLBACK;
+SELECT * FROM animals WHERE species <> 'unspecified'; 
+
+BEGIN;
+
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+SELECT * FROM animals WHERE species = 'digimon'; 
+
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+SELECT * FROM animals WHERE species = 'pokemon'; 
+
+COMMIT;
+
+BEGIN;
+DELETE FROM animals;
+ROLLBACK;
+
+
+SELECT * FROM animals
+
+BEGIN;
+SELECT * FROM animals
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT my_savepoint;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO SAVEPOINT my_savepoint;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+
+SELECT * FROM animals
+
+BEGIN;
+SELECT * FROM animals
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
```

Comment:
- [ ] As requested by the previous reviewer, kindly add screenshots to all your queries in the PR description. Your screenshot should show the query and the result you got from the query. I can only see 4 results out of the 11 queries, kindly add the rest of the screenshots.